cc3d/player5/Configuration/ui_configurationdlg.py

```diff
@@ -831,6 +860,17 @@ def retranslateUi(self, CC3DPrefs):
         self.useInternalConsoleCheckBox.setText(_translate("CC3DPrefs", "Use internal console for text output"))
         self.closePlayerCheckBox.setToolTip(_translate("CC3DPrefs", "<html><head/><body><p>If checked this will cause player close after simulation is done</p></body></html>"))
         self.closePlayerCheckBox.setText(_translate("CC3DPrefs", "Close Player after simulation ends"))
+        self.label_25.setText(_translate("CC3DPrefs", "Log Level"))
+        self.log_level_combo.setCurrentText(_translate("CC3DPrefs", "LOG_INFORMATION"))
```

Comment:
I think it'd be ok to list these without the "LOG_" prefix

Reply:
good point. I can fix that

Reply:
ok this is fixed now